fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -741,9 +742,9 @@ public LoanTransaction handleChargeAppliedTransaction(final LoanCharge loanCharg
             transactionDate = loanCharge.getDueLocalDate();
```

Comment:
Arent these changes irrelevant to FINERACT-1921? Please extract them into a separate PR if so!